Formula/phpunit@7.5.rb

```diff
@@ -0,0 +1,110 @@
+class PhpunitAT75 < Formula
+  desc "Programmer-oriented testing framework for PHP"
+  homepage "https://phpunit.de"
+  url "https://phar.phpunit.de/phpunit-7.5.2.phar"
+  sha256 "64682015804129b83e5dcfe5d1b9810d39f161a25ca1207ca8befe31204297d2"
+
+  bottle :unneeded
+
```

Comment:
Adding `key_only` line.
```suggestion

keg_only :versioned_formula

```